pillar/coalition.sls

```diff
@@ -2,6 +2,7 @@ backup:
   directories:
     # Must match directory in coalition/init.sls.
     /home/coalition/public_html/:
+    /home/corporate/public_html/:
```

Comment:
In principle, we can have a separate pillar/corporate.sls file (we have coalition.sls separate from cms.sls, to make it easier to e.g. delete an entire site).